microsoftgraph/microsoft-graph-explorer-v4
# Bug: Persisting request body

## Overview
Here is a replication of the bug:
* Fill in the request body and click "Run Query"
* Click on a sample query
* Click on "Run Query" and the previous request body that was inputted will show up again
* If there is a malformed request body, it will also persist and show the malformed error when you run the next query

### Demo
Here is a demo of the bug

https://user-images.githubusercontent.com/47228306/129066815-2075656c-ca79-4ce1-ad1c-699f277cf0c7.mp4

Here is a demo of the fix

https://user-images.githubusercontent.com/47228306/129066915-5aad819b-3899-4cee-98f9-ece51c6e8142.mp4


## Testing Instructions

* Fill in the request body and click "Run Query"
* Click on a sample query
* Click on "Run Query" and see that the previous request body doesn't show up anymore
